.github/workflows/pr-if-new-kernel.yml

```diff
@@ -0,0 +1,33 @@
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+      - run: |
+          OUTPUT=$(python update-kernel.py)
+          echo $OUTPUT
+          echo "has-update=$OUTPUT" >> "$GITHUB_OUTPUT"
+        id: update
+      - name: create pull request
```

Comment:
Will this create a PR every night it's running? Could it happen that when we're updating this on Saturday will come to 3 PR's on Monday?